packages/components/src/Actions/ActionSplitDropdown/ActionSplitDropdown.component.js

```diff
@@ -56,14 +59,14 @@ function ActionSplitDropdown(props) {
 			onClick={event => rClick(event, onClick, { label, ...rest }, model)}
 			title={Title}
 			id={uuid.v4()}
-			className={theme['tc-split-dropdown']}
+			className={classNames(className, theme['tc-split-dropdown'])}
 			{...rest}
 		>
 			{
 				items.length ?
 					items.map((item, index) => (
 						<MenuItem {...item} key={index}>
-							{ item.icon && <Icon name={item.icon} /> }
+							{ item.icon && showIcons ? <Icon name={item.icon} /> : null }
```

Comment:
this goes against what we have done in Action.
we display all the provided stuff